package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-actions/codebuild-run-build",
-  "version": "1.0.0",
+  "version": "1.0.3",
```

Comment:
Ouch, what happened here?

Reply:
The package lock json has never been updated.
Tools are better than humans :(